### DISCUSSION TITLE
Log Location

### DISCUSSION THREAD

**Discussion by savidudu** [keepassxreboot/keepassxc#8284] [Q&A]
I tried installing the KeePassXC and not able to see anything in the log location.

I mean the product gets installed successfully but when I am going to the log location, I am only seeing the folder but there is no log entry available.

Here is the location which I am referring:

**Linux**=/usr/share/keepassxc
**MAC**=/Applications/KeePassXC or app/Contents/Resources
**Windows**=C:\Program Files\KeePassXC or C:\Program Files (x86)\KeePassXC for 32 Bit 

The other query from my end:

The password we use to login to the keepassXC application , how is that encrypted - the method used ( AES-256 etc type) ??

Thanks

**Comment by droidmonkey** [comment]
I am not sure what you are trying to ask?

**Reply by savidudu** [reply]
Oops...
The question is Where we can see the installation logs ?
The Password we use to login in the application , How is that encrypted , I mean the encryption method ?

**Reply by droidmonkey** [reply]
Installation logs are wherever your operating system puts them. We don't manage Installation logs. 

Your actual password is hashed using SHA-256 and then transformed using a key derivation function (default Argon2) which makes brute forcing impossible. If you use a key file and/or YubiKey then that key material is hashed and serially added to your master key which is again hashed.
